adding-new-application/README.md

```diff
@@ -252,6 +252,19 @@ You can validate your changes locally using the `plural link` command.  You'll n
 plural link helm <app-name> --path ../path/to/helm --name <chart-name>
 ```
 
+Next step is to build and deploy with the new changes:
+
+```
+plural build --only <app-name> && plural deploy
+```
+
+To make sure the change detection is working properly, you can also use the --force option of plural build command
```

Comment:
small grammar fix:

To make sure change detection is working properly, add the `--force` flag to the `plural build` command

Reply:
Corrected. Thank you !